### PR TITLE
Adding code to convert all negatives values of Power up to 0 (first good issue #30)

### PR DESCRIPTION
**Pull request by DubraskaS**
# Pull Request

## Description

_Added code to move up to 0 all negative values of Power_

Fixes #

_Modified client.py in folder internal>inputs>indiadb adding conditional, and resample.py in folder internal>inputs>service using boolean indexing before assigning, regarding this issue ._

### CODE REVIEW

File: src/india_api/internal/service/resample.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def resample_generation(values: list[ActualPower], internal_minutes: int) -> lis
 
     # resample
     df = df.set_index("Time").resample(f"{internal_minutes}T").mean().dropna()
+    df.loc[df['PowerKW'] < 0, 'PowerKW'] = 0  # Set negative values of PowerKW up to 0
```
Comment (peterdudfield) [inline src/india_api/internal/service/resample.py:19]: Do you mind doing something like this 
`df['PowerKW'].clip(lower=0, inplace=True)`,
I think its slightly easier to read

**Review by peterdudfield — CHANGES_REQUESTED** [review]
Thanks for list, ive added one small comment, and then im sure we can get this merged

**Comment by DubraskaS** [comment]
Hi! Thank you for the feedback! I added the change and I agree, it's easier to read.

Push done in the repo linked to the PR.

**Comment by DubraskaS** [comment]
Yes! I apologize, I pushed into the main and not the branch of my contribution. Updated now, you should be able to see the changes in the PR.

**Review by peterdudfield — APPROVED** [review]

**Comment by peterdudfield** [comment]
@all-contributors please add @DubraskaS for code

**Pull request merged**